rfcs/0014-abi-errors.md

```diff
@@ -0,0 +1,456 @@
+- Feature Name: `abi_errors`
+- Start Date: 2024-11-18
+- RFC PR: [FuelLabs/sway-rfcs#43](https://github.com/FuelLabs/sway-rfcs/pull/43)
+- Sway Issue: [FueLabs/sway#0000](https://github.com/FuelLabs/sway/issues/001)
```

Comment:
Should there be a corresponding issue created on the sway repo?

Reply:
This comes at the end, when the RFC is accepted.